knowhere/knowhere/utils/CMakeLists.txt

```diff
@@ -14,34 +14,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set(UTILS_SRC
-        distances_simd.cpp
-        FaissHookFvec.cpp
-        )
+if (APPLE)
```

Comment:
to refer CMakeLists of root dir,  use MACOS as a variable.

Reply:
done.